docs/contributing.md

```diff
@@ -26,7 +145,7 @@ Those include:
   - `PulpEntityContext.pulp_href`: This property can be used to specify an entity by its URI.
     It will be fetched from the server only at read access.
 
-### Compatibility to pulp versions
+## Compatibility to pulp versions
```

Comment:
This one should probably be merged with "Version dependent codepaths". This one is the goal, the other one is the tooling for it.
We should have them adjacent at least.

Reply:
I have tried to do this @mdellweg let me know if you have any suggestions